enterprise-suite/tests/resources/gke-ingress.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: expose-es-console-ing
+spec:
+  backend:
+    serviceName: expose-es-console
+    servicePort: 80
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: expose-prometheus-ing
```

Comment:
Do these expose-X things need a `NodePort` specified?  I'm confused.  Weren't these added specifically because we needed to use a NodePort?  If so, where is the NodePort referenced?  Maybe I'm missing something.